scripts/test-fuzzed-MAC.py

```diff
@@ -47,12 +51,13 @@ def main():
         node = node.add_child(FinishedGenerator())
         node = node.add_child(ExpectChangeCipherSpec())
         node = node.add_child(ExpectFinished())
-        node = node.add_child(fuzz_mac(ApplicationDataGenerator(
-                                                        b"GET / HTTP/1.0\n\n"),
-                                       xors={pos:val}))
+        node = node.add_child( \
+            fuzz_seal(ApplicationDataGenerator( b"GET / HTTP/1.0\n\n"), xors={pos:val})
```

Comment:
duh, it won't work because it is applied pre-encryption, not post encryption

what I wanted to say is, why not apply fuzzing in the `send()` method of `RecordSocket`, and make a general `fuzz_ciphertext` method so that it is also applicable to Encrypt-then-MAC ciphers? `fuzz_seal` doesn't limit the modification to just the `tag` portion of the AEAD ciphertext...